README.md

```diff
@@ -7,3 +7,9 @@
 - `npm install`
 - `bower install`
 - `grunt build`
+
+## New Version of Stache
+
+We no longer support development on this version of Stache. Instead, we recommend you check out our latest version of Stache, the Blackbaud Documentation Builder. The currently supported version is a collection of Angular 2 components for SKY UX SPAs. 
+
+https://github.com/blackbaud/stache2
```

Comment:
What do you think of removing the URL here and make "our latest version of Stache" a link to the URL? I think that would look cleaner, but it's probably 6 of one, half dozen of the other.

Reply:
I hadn't really thought either way. Flip a coin. What sounds better to you?

Reply:
My preference would be to make "our latest version of Stache" the link. ... But I don't feel strongly about it.